code/script.js

```diff
@@ -1,13 +1,31 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const inputWrapper = document.getElementById('input-wrapper')
+const input = document.getElementById('input')
+const sendBtn = document.getElementById('send')
+const yesButton = document.getElementById('yes-btn')
+const noButton = document.getElementById('no-btn')
+const nameInput = document.getElementById('name-input')
 
-// Global variables, if you need any, declared here
+//Makes a function for showMessage 'user' (cleaner code)
+const userReply = (msg) => {
+  showMessage(msg, 'user');
+}
+//Makes a function for showMessage 'bot' (cleaner)
+const botReply = (msg) => {
+  showMessage(msg, 'bot');
+}
```

Comment:
You can try to put your timer function here to avoid repeating it over and over again 😸